cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -293,10 +293,10 @@ func getControllerManagerCommand(cfg *kubeadmapi.ClusterConfiguration, k8sVersio
 	// Let the controller-manager allocate Node CIDRs for the Pod network.
 	// Each node will get a subspace of the address CIDR provided with --pod-network-cidr.
 	if cfg.Networking.PodSubnet != "" {
-		maskSize := calcNodeCidrSize(cfg.Networking.PodSubnet)
+		maskSize := calcNodeCidrSize(cfg.Networking.PodSubnet) // TODO(awander): Needs to be fixed once PR #73977 lands
```

Comment:
I think that comments should go alone in the previous line, at least I find them easier to read